modules/page-edit.php

```diff
@@ -162,11 +167,11 @@
 					$page_tags = $_POST['tags'];
 
 				// Insert the "view" part of the page we're editing
-				$content .= parse_page_source($pagetext);
+				$content .=  "<h4>To continue editing or save, scroll down...</h4>" . parse_page_source($pagetext);

```

Comment:
In fact that's a better idea, thinking about it. That way a natural `<h4>` that's part of the page doesn't get the `text-align: center;` applied to it when we don't want it to be.